atest/testdata/standard_libraries/operating_system/get_file.robot

```diff
@@ -131,8 +141,11 @@ Grep File non Ascii
     [Setup]    Create File    ${TESTFILE}    fää\nbär\nföö bär\n\nA Fåå
     [Template]    Grep And Check File
     fää     fää        ${TESTFILE}
+    fää     fää        ${TESTFILE}    regexp=${True}
     ö       föö bär    ${TESTFILE}
+    .*ö     föö bär    ${TESTFILE}    regexp=${True}
     A       A Fåå      ${TESTFILE}
+    A       A Fåå      ${TESTFILE}    regexp=${True}
```

Comment:
1. Separate regexp test might be a good idea here as well.
2. Should test that `\w` matches non-ASCII characters. Possibly `fää` could be changed to `f\\wä` (note double `\\` needed by Robot's escaping rules).